apps/system/js/accessibility.js

```diff
@@ -45,6 +45,12 @@
     CONTRAST_CAP: 0.6,
```

Comment:
Overall in this file, lets use hints consistently (avoid using hint) in all names. We can have more than 1 hint per element so hints is more appropriate.